src/twisted/internet/test/test_filedescriptor.py

```diff
@@ -55,22 +55,6 @@ class FileDescriptorTests(SynchronousTestCase):
     Tests for L{FileDescriptor}.
     """
 
-    def test_writeWithUnicodeRaisesException(self):
```

Comment:
Please restore these tests; the implementation here definitely shouldn't expect `str`.